src/include/meson.build

```diff
@@ -1,7 +1,7 @@
 configure_file(output: 'config.h', configuration: cdata)
 
 src_lib_common += vcs_tag(
-  command: './git_tag.sh',
+  command: [python_path, src_root / 'version.py'],
```

Comment:
You use the path to python here, but command lists are supposed to accept found programs here almost per definition, because every command begins with a program (and continues with optional args).

Reply:
Well, it doesn't here… tried that, of course.